### ISSUE TITLE
Trying Login With Google

### ISSUE THREAD

**Issue by MrCrunchy** [pokesniperandroid/PokeSniper-Android#188]
Seems to stay stuck at times and simply states:

WAITING SNIPE

Ready!
Trying login with Google. . .

Then nothing. Restart the app and try again and it occasionally works and occasionally it's this screen again.

**Comment by Herng97** [comment]
Same!

**Comment by chimsedinang123** [comment]
Me to

**Comment by sapukasibersih** [comment]
Me too

**Comment by xiaoming93** [comment]
Yeah and it can happen at any point before catching the pokemon.  For example using pinap Berry and it stuck there forever.  Worst I had was restarting multiple times for 13 mins to catch a 100iv hitmonchan till it despawned haha

**Comment by Herng97** [comment]
when you're stuck at using Pinab berry.Sometime the pokemon is already caught but stucked there lol

**Comment by pokesniperandroid** [comment]
If the operation took too long to finish, either the connection is unstable or pokehash server is slow when giving response (but the request still sent - sometimes). This happened on PTC as well, as we test it before. Maybe we'll implement timeout setting (we don't know if that could help) but we'll try.

When we did tests: 4 minutes just to login, not doing snipe.

**Comment by xeliman** [comment]
I have the same problem, 1 week ago it works without problems, snipe the pokemon in a moment

**Comment by Yashhhh** [comment]
or maybe 150 users per minute thing is the culprit?

**Comment by Yashhhh** [comment]
this problem has started to take more often all of a sudden... dont know why.. :/

**Comment by MrCrunchy** [comment]
I agree. Was the cause of the error discovered?  It's become almost every snipe.  Restarting after a few minutes initially worked but even that no longer seems to help.

**Comment by pokesniperandroid** [comment]
Still testing.

**Comment by kabirash** [comment]
Same issue here. Once i force stop a few times and then it works again but not for long couple of snipes then freezes again.

**Comment by pokesniperandroid** [comment]
Sorry for the long update, test BETA11 with updated API.

**Comment by kabirash** [comment]
Thanks new upgrad is workinga treat

**Comment by anyDroidzOS** [comment]
same for me w/ PTC and Google, too.
but it freeze mostly after successfully login. on mobile networks and wifi the same problem, but i must say my datatraffic from my internet flat are "empty"/or must i really take an bigger internetflat? or any solutions to get it wrk fine on wifi? puuh, what an long txt, sorri guys^^

, to my english, i hope it's correctly written.
im german, w/ english attributes^^ hehe
